dvc/repo/__init__.py

```diff
@@ -334,7 +334,17 @@ def _collect_graph(self, stages=None):
             for out in stage.outs:
                 for p in out.path_info.parents:
                     if p in outs:
-                        raise OverlappingOutputPathsError(outs[p], out)
+                        msg = (
+                            "Paths for outs:\n'{}'('{}')\n'{}'('{}')\noverlap."
+                            " To avoid unpredictable behaviour, "
```

Comment:
Please use consistent formatting. You skip trailing space in the first line but add it in the second one. IIRC prefered way to have `str ` and not ` str`

Reply:
Agree about consistency. Idk what's preferred though, I used to put trailing space last but later I noticed most of you use trailing first so rn there's a mix throughout docs. Maybe trailing space first is best because it makes it extra obvious that it's a continuation string? Up to you, I just rec to be consistent (available to review all existing strings once you decide, please lmk).